app/addons/cors/components/CORSScreen.js

```diff
@@ -0,0 +1,150 @@
+import React, { Component } from "react";
+import app from "../../../app";
+import ReactComponents from "../../components/react-components";
+import FauxtonComponents from "../../fauxton/components";
+import Origins from "./Origins";
+import OriginInput from "./OriginInput";
+import OriginTable from "./OriginTable";
+
+const LoadLines = ReactComponents.LoadLines;
+const ConfirmationModal = FauxtonComponents.ConfirmationModal;
+
+export default class CORSScreen extends Component {
+
+  constructor (props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    // const { dispatch, url } = this.props;
+    this.props.fetchAndLoadCORSOptions();
+  }
+
+  enableCorsChange () {
+    console.log("CORSScreen.enableCorsChange:", this.props.corsEnabled, 'to', !this.props.corsEnabled);
```

Comment:
Its a little neater to rather do this at the begining of the function:

`const {corsEnabled, origins, node} = this.props`